Views/backend/connect/controller/main.js

```diff
@@ -1427,6 +1430,29 @@ Ext.define('Shopware.apps.Connect.controller.Main', {
 
     },
 
+    onResetPriceType: function () {
+        var me = this;
+
+        Ext.Ajax.request({
+            scope: this,
+            url: '{url module=backend controller=ConnectConfig action=resetPriceType}',
+            success: function (result) {
+                var response = Ext.JSON.decode(result.responseText);
+                if (response.success) {
+                    me.createGrowlMessage(
+                        me.messages.priceResetLabel,
+                        me.messages.priceResetSuccess
+                    );
+                } else {
+                    me.createGrowlMessage(
+                        me.messages.priceResetLabel,
+                        response.message
```

Comment:
This error is not translated. It comes from SN. Do we need to show some generic message and real reason to be in the logs?